lib/src/10_auth/auth_screen.dart

```diff
@@ -128,10 +146,45 @@ class _FauiAuthScreenState extends State<FauiAuthScreen> {
     );
   }
 
-  Widget _buildCreateAccountScreen(BuildContext context) {
-    final TextEditingController emailController =
-        new TextEditingController(text: this._email);
+  Widget _buildTextBox(
+      TextEditingController controllerName,
+      FocusNode currentNode,
+      FocusNode nextNode,
+      String fieldName,
+      bool autoFocus,
+      Future<Widget> Function() submit) {
+    handleKey(RawKeyEvent key) {
+      if (key is RawKeyDownEvent) {
+        if (key.physicalKey.debugName == 'Enter') {
+          submit();
+        } else {
+          {
+            setState(() {
+              this._error = "";
+            });
+          }
+        }
+      }
+      if ((key.logicalKey.debugName == 'Tab') && (nextNode != null)) {
+        nextNode.requestFocus();
+      }
+    }
 
+    return RawKeyboardListener(
+        focusNode: FocusNode(),
+        onKey: (key) => handleKey(key),
+        child: TextFormField(
+          focusNode: currentNode,
+          controller: controllerName,
+          autofocus: autoFocus,
+          onEditingComplete: () => {},
```

Comment:
Please, add comment why we need this statement, so that future contributors does not get confused.

Reply:
done